### PR TITLE
Ceaser Cipher Conversion Project

### PR DESCRIPTION
**Pull request by Damini2004**
## Pull Request for WebMasterLog 💡
Ceaser Cipher Conversion Project
### Issue Title: Ceaser Cipher conversion

- **Name of the project** : Ceaser Cipher Conversion Project
- **Category:**: Frontend Website
- **Name:** Damini Chachane
- **GitHub ID:** https://github.com/Damini2004
- **Idenitfy yourself:** GSSOC24

Closes: #124 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
I have added the algorithm via which we can easily convert the text.

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
dark
light

Uploading freecompress-Caesar Cipher and 7 more pages - Personal - Microsoft_ Edge 2024-05-12 04-14-06.mp4…

### Note to reviewers 📄
I generally designed the UI of ceaser cipher in basic format because the main concept behind the ceaser cipher is its conversion of text as per the keys. I posted commit in Frontend intermediate project.Kindly gothrough and suggest any improvement.

### CODE REVIEW

File: Front-end-Projects/Intermediate/Ceaser Cipher/ceaser_cipher.html
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Ceaser Cipher/ceaser_cipher.html:1]: Format the code properly of all the files.

File: Front-end-Projects/Intermediate/Ceaser Cipher/cipher.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    width: 100%;
+    padding: 10px;
+    margin-bottom: 10px;
+    border: 1px solid black; /* Dark border color */
+    border-radius: 5px;
+    box-sizing: border-box;
+    background-color: white; /* Dark input background color */
+    color: black; /* Light text color */
+  }
+  /* textarea{
```
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Ceaser Cipher/cipher.css:73]: Remove any un-necessary code.

File: Front-end-Projects/Intermediate/Ceaser Cipher/main.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// main.js
+const form = document.querySelector('form');
+const plaintextInput = document.querySelector('textarea[name="plaintext"]');
+const shiftInput = document.querySelector('input[name="shift"]');
+const outputDiv = document.querySelector('#output');
+
+const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+
```
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Ceaser Cipher/main.js:8]: remove any un-necessary new lines.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Damini2004 I think `cipher.css` and `style.css` have same code.
So, remove 1 file.
And follow the structure and guidelines mentioned in `CONTRIBUTING.md` file.
If you feel any queries, go through successful merged prs of `Front-end-Projects` OK.

**Comment by Damini2004** [comment]
Hello sir @Avdhesh-Varshney  No sir Style.css and cipher.css contain different program that if user want dark theme then style.css will run and if user want light theme cipher.css will run.
As you mentioned to format the html document,and rremove unnecessary javascript program i have resolve that issue

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Damini2004 Everything is perfect.
But i have told for reviewing because you have not read guidelines properly.
- Change directory `Ceaser-Cipher`.
- And add a screenshot with extension of webp as `screenshot.webp`.
- Also you have included a `README.md` file.

Now, if you have any query, either go through merged PR's or tag me in GSSOC server.

**Comment by Damini2004** [comment]
Ceaser Cipher Conversion Project #127

File: Front-end-Projects/Intermediate/Ceaser Cipher/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Ceaser Cipher/README.md:1]: Follow this [README TEMPLATE](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/.github/PROJECT-README-TEMPLATE.md).

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Damini2004 
- See this PR is perfect [LINK](https://github.com/Avdhesh-Varshney/WebMasterLog/pull/131/files)
- I have provided you name with a hyphen and told you to read guidelines carefully.
- Still you have not attach 1 screenshot which have extension of webp format.

**Comment by Avdhesh-Varshney** [comment]
@Damini2004 There is a suggestion to you. 
- If time exceedes more than 24 hrs, drop a msg in GSSOC server to either mentors or project admin (me) and 
- review the edited template of PR what i have changed for future contributions.

And there is a request to you, because of my fault, i had attach the front-end-projects label with this project but it is a vanilla-js-project. So, cut paste the project and resolve the above errors, commit all the changes in 1 go.

**Comment by Damini2004** [comment]
Kinldy add GSSOC24  label with level that will be more beneficial for me under GSSOC Contributor.

**Pull request closed without merging**